docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
+You can use a migration wave for creating Jira issues automatically for each application assigned to it.
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. From the *Options* menu (three dots) of the migration wave for which you want to create Jira issues, select *Export to Issue Manager*.
++
+The *Export to Issue Manager* window opens.
+. Select the instance type (Jira Cloud or Jira Server/Datacenter).
+. Select the instance, project and issue type from the drop-down lists.
```

Comment:
"...instance, project, and issue type from the lists." More than one list, correct? I can't see the UI, so pardon "obvious" questions.  IBM says not to use "drop-down" or any other modifiers for lists, unless more than one type of list is present at the part of the UI (I paraphrase).